py/desitarget/data/targetmask.yaml

```diff
@@ -144,6 +144,14 @@ obsmask:
     - [MORE_ZGOOD,  4, "redshift known; need more observations"]
     - [DONOTOBSERVE,  16, "Do not observe this target (possibly temporarily)"]
 
+#- Bits that can be set in TARGETID. See the schema at
+#- https://desi.lbl.gov/DocDB/cgi-bin/private/RetrieveFile?docid=2348
+targetid_mask:
+    - [BRICKID,    22, "RIGHTMOST bit that indicates BRICKID (BRICKID encompasses bits 22-41 inclusive"]
+    - [DR,         42, "RIGHTMOST bit that indicates Data Release (DR encompasses bits 42-47 inclusive"]
+    - [MOCK,       48, "this target is a mock target"]
+    - [SKY,        49, "this target is a sky fiber target"]
```

Comment:
Might as well include `OBJID` in here as well to give the complete specification of targetid bits.  Clarify whether you are counting from 0 or 1 (I hope zero).  Should this be encoded as `[NAME, FIRSTBIT, NBITS, "Description"]` ?  That over constrains the problem, but might be easier for code to deal with (e.g. it doesn't have to know that `DR` comes after `BRICKID` and use the combination of them to figure out how many bits are reserved for `BRICKID`.  A unit test could confirm that all bits are accounted for and non-overlapping (especially if you also add a `RESERVED` section at the end).